### ISSUE TITLE
Can we have a provision to reuse the reflatent?

### ISSUE THREAD

**Issue by kris775** [brianfitzgerald/style_aligned_comfy#13]
Can we save the reference latent generated by the node "**Style aligned sample reference latent'** for one style and then reuse it again in **"Style aligned reference sampler"**?. 
If the processing of reference image need not be done again for each run when using the same reference image  then saving and reusing might help in reducing the time of each run a bit and can help in a continuous iteration type of workflow.

**Comment by jasonphillips** [comment]
Do you mean saving it offline / on disk for future use, or do you just mean caching the result between runs? 

It should already cache and not require a re-run between generations if you use a fixed seed for it and the same image input / prompt input to the node, so perhaps I'm misunderstanding the request.

**Comment by dfl** [comment]
that idea reminds me a bit of this feature... saving embeds for IP adapter https://youtu.be/mJQ62ly7jrg?si=V37H7HMK_KeymnPl&t=846

**Comment by gseth** [comment]
> Do you mean saving it offline / on disk for future use, or do you just mean caching the result between runs?
> 
> It should already cache and not require a re-run between generations if you use a fixed seed for it and the same image input / prompt input to the node, so perhaps I'm misunderstanding the request.

What would help in batch align, not sure if this is possible.

im1 ( first as reference) - img2 - img3 - img4

Now I need say around 30 images, Due to vRAM limitation, everytime I have to regenerate in batches of 4. the first one again being the same image, since I only want img2, 3, 4.

If there would be a way to save the image 1, and keep generating only 3 more images on every queue prompt, it would be great.

**Comment by kris775** [comment]
> Do you mean saving it offline / on disk for future use, or do you just mean caching the result between runs? 

Yes i mean saving it offline on disk and reusing later. its like having pre saved  different styles which can be directly used, similar to saving embeds for the IP adapter.
